pkg/controllers/networkinfo/networkinfo_controller.go

```diff
@@ -77,8 +77,15 @@ func (r *NetworkInfoReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		if !isShared {
 			err = r.Service.CreateOrUpdateAVIRule(createdVpc, obj.Namespace)
 			if err != nil {
-				log.Error(err, "operate failed, would retry exponentially", "NetworkInfo", req.NamespacedName)
-				updateFail(r, &ctx, obj, &err, r.Client)
+				state := &v1alpha1.VPCState{
+					Name:                    *createdVpc.DisplayName,
+					VPCPath:                 *createdVpc.Path,
+					DefaultSNATIP:           "",
+					LoadBalancerIPAddresses: "",
+					PrivateIPv4CIDRs:        []string{},
```

Comment:
privateCIDRs need be set

Reply:
For VPC update case, we need get DefaultSNATIP/LoadBalancerIPAddresses/PrivateIPv4CIDRs here, right?

Reply:
yes, this part is for failed case, we updated what we can get to NetworkInfo CR and put the event back to queue for next loop.